### ISSUE TITLE
[VOLT] Support for embedding blocks into other blocks

### ISSUE THREAD

**Issue by applejohnny** [phalcon/phalcon#133]
With inheritance of multiple files like a.volt extends b.volt extends c.volt, I have to use embedding blocks. Otherwise there's no profit to use inheritance more than one nesting level.

**Comment by vladimmi** [comment]
Stale bots are the plague of GitHub making any inactivity a sign of "everything is great now" and closing unresolved issues...

**Comment by niden** [comment]
> Stale bots are the plague of GitHub making any inactivity a sign of "everything is great now" and closing unresolved issues...

Yeah, after today I am considering deactivating it. I had to remove the stale label from over 20 issues so far :/
